src/main/java/wooteco/subway/admin/domain/LineStation.java

```diff
@@ -17,6 +20,41 @@ public LineStation(Long preStationId, Long stationId, int distance, int duration
         this.duration = duration;
     }
 
+    public boolean isStationIdNull() {
```

Comment:
```suggestion
    public boolean haveStation() {
```
```suggestion
    public boolean containStation() {
```

너무 코드레벨에 가까운 이름으로 보여집니다!
해당 메서드가 어떠한 정보를 나타내는지 고민해보면 좋을 것 같아요!